docs/en/query_language/select.md

```diff
@@ -448,6 +448,19 @@ The table names can be specified instead of `<left_subquery>` and `<right_subque
 
 See the standard [SQL JOIN](https://en.wikipedia.org/wiki/Join_(SQL)) description.
 
+#### Syntax Limitations
+
+For multiple `JOIN`:
```

Comment:
Would it better to place it after Multiple JOIN section?

Reply:
![image](https://user-images.githubusercontent.com/2968367/58419656-e01c2700-8093-11e9-8d59-197f3e7d1a14.png)
After the Multiple `JOIN`  section it will look not in place, I think. We can place it before Usage Recommendations as the end of syntax topics.